parser-typechecker/unison/Main.hs

```diff
@@ -172,7 +178,7 @@ main = do
       case e of
         Left _ -> PT.putPrettyLn $ P.callout "⚠️" "I couldn't find that file or it is for some reason unreadable."
         Right contents -> do
-          (closeCodebase, theCodebase) <- Codebase.getCodebaseOrExit cbInit mcodepath
+          (closeCodebase, theCodebase) <- getCodebaseOrExit cbFormat mcodepath
```

Comment:
Hang on to the choice of which `Codebase.Init` to use until later in `getCodebaseOrExit`, so that we can try more than one if appropriate